tests/example/index.rst

```diff
@@ -19,4 +19,13 @@ Example 2
    :gridsize: 20.
    :griddivisions: 20
 
-   mypart = cadquery.Workplane("XY").box(2, 2, 2)
\ No newline at end of file
+   mypart = cadquery.Workplane("XY").box(2, 2, 2)
+
+Example 3
+^^^^^^^^^
+
+.. cadquery::
+   :select: mypart
+   :gridsize: 0
+
+   result = cadquery.Workplane("XY").box(2, 2, 2)
```

Comment:
You are selecting `mypart` but the variable is `result`. I guess this may throw an error?